pkg/logs/internal/tailers/file/tailer.go

```diff
@@ -20,38 +20,33 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/containers"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 
-	"github.com/DataDog/datadog-agent/pkg/logs/config"
 	"github.com/DataDog/datadog-agent/pkg/logs/decoder"
 	"github.com/DataDog/datadog-agent/pkg/logs/message"
-	"github.com/DataDog/datadog-agent/pkg/logs/parser"
 	"github.com/DataDog/datadog-agent/pkg/logs/tag"
 )
 
-// DefaultSleepDuration represents the amount of time the tailer waits before reading new data when no data is received
-const DefaultSleepDuration = 1 * time.Second
-
 // Tailer tails one file and sends messages to an output channel
 type Tailer struct {
 	readOffset    int64
 	decodedOffset int64
 	bytesRead     int64
 
-	// file contains the logs configuration for the file to parse (path, source, ...)
-	// If you are looking for the os.file use to read on the FS, see osFile.
-	file *File
+	// File contains the logs configuration for the File to parse (path, source, ...)
+	// If you are looking for the os.File use to read on the FS, see osFile.
+	File *File
 
 	fullpath string
 	osFile   *os.File
 	tags     []string
 
-	outputChan  chan *message.Message
+	OutputChan  chan *message.Message
```

Comment:
were these properties made public just to satisfy unit tests?

Reply:
I think so.  We can clean that up in the followup PRs.

Reply:
ok just don't want to lose track of it.